pabi_base/models/product.py

```diff
@@ -19,6 +19,7 @@ class ProductTemplate(models.Model):
         readonly=False,
         size=10,
     )
+    valuation = fields.property(track_visibility='onchange',)
```

Comment:
I am not sure there is `fields.property` in Odoo 8. I think fields.property is from Odoo 7 isn't it?